### ISSUE TITLE
Class 'Imtigger\LaravelJobStatus\LaravelSettingsUIServiceProvider' not found

### ISSUE THREAD

**Issue by jamestowers** [imTigger/laravel-settings-ui#1]
After adding 

```
Imtigger\LaravelJobStatus\LaravelSettingsUIServiceProvider::class
```

to my providers array in L5.4 I'm getting the class not found error when I run the vendor:publish command

**Comment by jamestowers** [comment]
Found the solution, the class path should be:
```
Imtigger\LaravelSettingsUI\LaravelSettingsUIServiceProvider::class
```

**Comment by imTigger** [comment]
Yes, my bad :/
Thank you for correcting this

**Comment by imTigger** [comment]
Fixed in f80bfeb2916e9b502fb76ccd16bd50b600c75055
